lib/resque_bus/server.rb

```diff
@@ -22,6 +22,123 @@ def self.included(base)
         
       }
     end
+
+    class Helpers
+      class << self
+        def parse_query(query_string)
+          has_open_brace = query_string.include?("{")
+          has_close_brace = query_string.include?("}")
+          has_multiple_lines = query_string.include?("\n")
+          has_colon = query_string.include?(":")
+          has_comma = query_string.include?(",")
+          has_quote = query_string.include?("\"")
+
+          exception = nil
+
+          # first let's see if it parses
+          begin
+            query_attributes = JSON.parse(query_string)
+            raise "Not a JSON Object" unless query_attributes.is_a?(Hash)
+          rescue Exception => e
+            exception = e
+          end
+          return query_attributes unless exception
+
+          if query_attributes
+            # it parsed but it's something else
+            if query_attributes.is_a?(Array) && query_attributes.length == 1
+              # maybe it's the thing from the queue
```

Comment:
Could you expand this comment to explain what "the thing from the queue" is? It sounds like a horror movie.